contracts/math/decimal.ral

```diff
@@ -169,8 +175,8 @@ Abstract Contract Decimal() {
         if (tickIndex >= 0i) {
             return rescale(sqrtPrice, FixedPointScale, SqrtPriceScale)
         } else {
-            let intermediate = one(SqrtPriceScale) / sqrtPrice 
-            return rescale(intermediate, FixedPointScale, SqrtPriceScale)
+            let intermediate = uints.bigDiv512(uints.toU512(one(SqrtPriceScale)), sqrtPrice, 1) 
```

Comment:
Do you need to use the U512 type here?

Here are actually these operations:
`fixed_point_denominator * fixed_point_denominator / sqrt_price_fixed_point_decima`

Reply:
currently we dont have U256 / U256 function, we only have U512 / U256, ill update it in the future when div refactor will be merged